### PR TITLE
Change ref name used in git to compute diff to "AURUTILS_SEEN".

### PR DESCRIPTION
**Pull request by reyreaud-l**
As discussed in #1049, change the ref name we use in git to compute whether or not to display diff to AURUTILS_SEEN.
Also introduce a global constant in download.go to avoid re-typing the string every time.

This is made in an effort to use the same name as aurutils, so that if one user decide to share the cache directory between different aur helpers, the tools are used the same ref to display diffs.

### CODE REVIEW

**Comment by Morganamilo** [comment]
I'd prefer just `AUR_SEEN`. Ideally I could convince @AladW to change it ;)

**Comment by reyreaud-l** [comment]
That would make more sense. Maybe merge it with AURUTILS_SEEN and wait until we agree with aurutils to change it ? Then we all change it at once ?

Otherwise I can change it to AUR_SEEN and update the PR.

**Comment by AladW** [comment]
If more than 1 project uses `AUR_SEEN`, I don't mind changing it in aurutils as well.

**Comment by Jguer** [comment]
`AUR_SEEN` or `AUR_REVIEWED` are equally good for me.

@AladW  Off-topic but it's already usual for yay topics: I understand that the default directory for repositories in `aurutils` is `~/.cache/aurutils/sync/` depending as well on`$XDG_CACHE_HOME` and `$HOME`.
If we standardized on a cache suffix (`${cache}/aur/sync`? `${cache}/aur`?) for AUR repos it could also benefit this change in diff ref names.
(Any other ideas on standardization of protocol or directories I'm interested)

**Comment by AladW** [comment]
For cache directories, aurutils has used `AURDEST` since the beginning, because `pacaur` was using the same variable. So you could define XDG_CACHE_HOME and HOME, or just set `AURDEST=$cache/aur/`, for example.

**Comment by Morganamilo** [comment]
Maybe we should respect AURDEST. not sure how I feel about using the same dir by default if that's what @Jguer is suggesting.

@E5ten

**Comment by E5ten** [comment]
I'll implement whatever name gets decided here for increased compatibility, AUR_SEEN seems like a fine choice.

**Comment by rafasc** [comment]
@Morganamilo Doesn't sound like a good idea at first. For example, `aurutils` will `reset --hard` repos in its operation and allows keeping local commits by pulling with `--rebase`. Having repos shared between tools kind of restricts a bit what you can do with the repos without stepping on each others toes.

**Comment by Jguer** [comment]
> Maybe we should respect AURDEST. not sure how I feel about using the same dir by default if that's what @Jguer is suggesting.
> 
> @E5ten

I think we should respect `AURDEST`, possibly ignoring config file settings if `AURDEST` is set even, could be useful for scripting and whatnot.
 
Merging default paths, could be nice but I see the possible pains of forced migration to users. @Morganamilo thoughts on the cons of this?

File: download.go [hunk 1]
```diff
@@ -14,6 +14,10 @@ import (
 	"github.com/Jguer/yay/v9/pkg/types"
 )
 
+const (
+	gitDiffRefName = "AURUTILS_SEEN"
```
Comment (Jguer) [inline download.go]: Just change this to `AUR_SEEN` and we're good

Comment (Morganamilo) [reply download.go]: And ditch the unneeded brackets.

**Pull request merged**